server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -51,6 +51,8 @@ case $opts in
         ;;
 esac
 
+TMP_PATH=$1
```

Comment:
Even though in some cases, the tarball will be unpacked in the TMP directory, in most other cases the unpacking will happen directly into the final destination, so TMP_PATH should probably be called UNPACK_PATH.